packmanager/umbrella.go

```diff
@@ -123,6 +126,55 @@ func (u umbrella) AddSource(ctx context.Context, source string) error {
 	return nil
 }
 
+func (u umbrella) Prune(ctx context.Context, qopts ...QueryOption) error {
+	var query Query
+	var tarFiles []fs.DirEntry
+	var errList []error
+	homeDir, err := os.UserHomeDir()
+	sourcesDir := homeDir + "/.local/share/kraftkit/sources/"
+	for _, qopt := range qopts {
+		qopt(&query)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	if query.all {
+		if err != nil {
+			return err
+		}
+		tarFiles, err = os.ReadDir(sourcesDir)
+		if err != nil {
+			return err
+		}
+
+		for _, tarFile := range tarFiles {
+			if tarFile.Name() == ".unikraft" {
+				continue
+			}
+			err := os.Remove(sourcesDir + tarFile.Name())
+			if err != nil {
+				errList = append(errList, err)
+			}
+		}
```

Comment:
Prune should only remove the sources directory, not the project-level build artifacts, so this can be removed.

Reply:
Done 👍🏽